### PR TITLE
Added canvas download button

### PR DESCRIPTION
**Pull request by aksh-hash**
changes made to files:
rectangle.html , circle.html , simplePen.html , ecllipse.html

### CODE REVIEW

**Comment by ExpressHermes** [comment]
You have added download button, right?

**Comment by ExpressHermes** [comment]
@aksh-hash Can you comment pics of downloaded canvases, please so that I can review them too?

**Comment by aksh-hash** [comment]
Sure.
canvas(4)
example(2)
example(1)
example

**Comment by aksh-hash** [comment]
Yes Sir.

**Comment by ExpressHermes** [comment]
@aksh-hash These images have a transparent background. Can you figure out a way to add a background(same as the background color of our canvas) to these images?

File: circle.html [hunk 1]
```diff
@@ -125,6 +125,17 @@
       }
     })
   </script>
+</script>
+<div>
+  <script>
+  function createDownload() {
+      const downloadURL = document.getElementById('myCanvas').toDataURL();
+      console.log(document.getElementById('myCanvas'));
```
Comment (TsarPrince) [inline circle.html:133]: @aksh-hash you forgot to remove this console log statement.

Comment (aksh-hash) [reply circle.html:133]: Sure, i'll do it. Thankyou

**Comment by aksh-hash** [comment]
@ExpressHermes , since canvas image is saved on the pixels, my approach was to first fill the canvas background with the desired color , and then save the image , but even after that its still the same. 
and the color of the canvas background is restored when the image is opened in Firefox or Google Chrome or any other web browser .
image

File: rectangle.html [hunk 2]
```diff
@@ -128,6 +128,17 @@
       }
     })
   </script>
+</script>
+<div>
+  <script>
+  function createDownload() {
+      const downloadURL = document.getElementById('myCanvas').toDataURL();
+      console.log(document.getElementById('myCanvas'));
```
Comment (ExpressHermes) [inline rectangle.html:136]: remove console statement

File: simplePen.html [hunk 3]
```diff
@@ -226,6 +226,17 @@
     })
 
   </script>
+</script>
+<div>
+  <script>
+  function createDownload() {
+      const downloadURL = document.getElementById('myCanvas').toDataURL();
+      console.log(document.getElementById('myCanvas'));
```
Comment (ExpressHermes) [inline simplePen.html:234]: same as above

File: ellipse.html [hunk 4]
```diff
@@ -129,6 +129,17 @@
       }
     })
   </script>
+</script>
+<div>
+  <script>
+  function createDownload() {
+      const downloadURL = document.getElementById('myCanvas').toDataURL();
+      console.log(document.getElementById('myCanvas'));
```
Comment (ExpressHermes) [inline ellipse.html:137]: same as above

File: simplePen.html (hunk 3 above)
Comment (aksh-hash) [reply simplePen.html:234]: sure

File: ellipse.html (hunk 4 above)
Comment (aksh-hash) [reply ellipse.html:137]: sure

**Review by ExpressHermes — COMMENTED** [review]
Please push the new commit in the same branch as this PR to update the code. It's still same as before

**Comment by aksh-hash** [comment]
ok

**Review by ExpressHermes — APPROVED** [review]

**Comment by aksh-hash** [comment]
Sir, I've found a way to save the image with the background color.
myImage

**Comment by ExpressHermes** [comment]
@aksh-hash Make a PR

**Pull request merged**